docs/docs/guides/database/03-orm/02-crud-operations.md

```diff
@@ -150,7 +150,7 @@ await User.query().where('is_verified', false).delete()
 ## Find Or Create
 Models comes a lot of helpful methods to simplify the record creation by first finding them inside the database and performing the create query, only when the record doesn't exists.
 
-### `firstOrCreate`
+### `findOrCreate`
```

Comment:
Actually, the method name is `firstOrCreate`. The code example is incorrect. Mind changing that?

Reply:
Not at all. Will do.